Automattic/jetpack
# Responsive Videos: support more embed block formats

#### Changes proposed in this Pull Request:
﻿
See https://github.com/Automattic/wp-calypso/issues/55357

Before WP 5.6 and this PR: https://github.com/WordPress/gutenberg/pull/24090
WordPress had multiple embed blocks, one for each service, all starting with core-embed/. After WP 5.6, we now have a single embed block, core/embed, with variations for each service.

Let's support both formats in the Responsive Videos.

#### Jetpack product discussion

No

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

* Start by enabling a theme that supports core Responsive embeds, like Twenty Nineteen.
* In a new post, paste a YouTube video; it will be automatically converted into an embed block with the video. Set that embed to a full width one.
* Publish your post.
* On the frontend, you should not see any jetpack-video-wrapper div around the embed. The video should take the full width of your browser.
* Edit that post. Delete that embed block, and instead insert a classic block. Inside that classic block, paste the same video URL.
* Save your changes.
* On the frontend, you should now see the jetpack-video-wrapper div around the embed.
